protocol/scripts/deploys/helpers.ts

```diff
@@ -35,10 +35,16 @@ export interface DeployedContracts {
   TEMPLE_TEAM_EPOCH_5: string;
   TEMPLE_TEAM_EPOCH_6: string;
   TEMPLE_TEAM_EPOCH_7: string;
+  TEMPLE_TEAM_EPOCH_8: string;
 
```

Comment:
i think you should add the team payments factory and implementation to the interface

Reply:
editing this file was causing conflicts so ill rebase this pr on latest stage branch to avoid those